ironfish/src/memPool/feeEstimator.ts

```diff
@@ -128,18 +138,35 @@ export class FeeEstimator {
     return feeRates.sort((a, b) => (a > b ? 1 : -1))
   }
 
-  estimateFeeRates(): { low: bigint; medium: bigint; high: bigint } {
+  private async getLastBlockSizeBytes(): Promise<number> {
```

Comment:
Could we update `latestBlockSize` as blocks come in through `onConnectBlock` and `onDisconnectBlock`?

It would be nice not to have to use the chain to look up blocks

Reply:
I thought about it, its easy to add in onConnectBlock. But for onDisconnectBlock,  i dont have a new value to update this latestBlockSize. In the end, I still need to keep this lastBlockSize check in case onDisconnect delete the value, and it becomes null.

Reply:
One way to address this might be to keep the block sizes in a queue the way we do with the `FeeRateEntry` queues.